.gitignore

```diff
@@ -1,4 +1,3 @@
 coverage/
 node_modules/
-wires
```

Comment:
wires isn't generated by anything in this project.

We used to download and name a file wires for selenium, then selenium looked for 'geckodriver' which we did but saved somewhere else and now we just use the geckodriver module.